tst/com/amazon/corretto/crypto/provider/test/TestUtil.java

```diff
@@ -60,10 +71,31 @@ public class TestUtil {
         new String[] {"secp256k1", "1.3.132.0.10"},
       };
 
-  public static final boolean isFips() {
+  public static boolean isFips() {
```

Comment:
why was the `final` qualifier removed here?

Reply:
It's redundant for static methods.